### ISSUE TITLE
SimNibs Crashes when opening ernie folder but not sphere

### ISSUE THREAD

**Issue by cesargutierrez7** [simnibs/simnibs#158]
Hello,

My Simnibs is crashing while trying to pull up the ernie head mesh. But  the sphere file is okay when i do run it.
![image](https://github.com/simnibs/simnibs/assets/137856269/631d4969-66d2-4a66-97d6-4e76c094bca2)

My PC running this has the latest CPU and GPU so it is more than capable to run this program. I may be doing something incorrect please help!

Thanks!

**Comment by oulap** [comment]
Are you using the example data from a previous simnibs version (3.x) or the new example data? The output folder structure is not compatible between simnibs versions, which might cause the crash.

**Comment by cesargutierrez7** [comment]
I am using the data downloaded from the simmons webpage which I think it might be 4.0

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by oulap** [comment]
Yeah it seems that's the correct example data. In the above screenshot you're trying to open the MNI152 example. Are you trying to open it by pointing it to the m2m-folder or the head mesh?

I could not reproduce this on a windows computer, so I'm not exactly sure what goes wrong.

**Comment by alwinzh** [comment]
I have met exactly the problem with v4.01 running on a windows 11 computer with AMD CPU and GPU, the usage of 24GB memory reached about 99%. But it works well on a Surface pro 8 with Intel CPU and GPU.  Is it possible that it is related to the AMD CPU or GPU?

**Comment by oulap** [comment]
Is the surface pro also win 11? We haven't tested on windows 11 (probably should), so I'm interested to hear if the operating system on both of those machines is the same. If yes, then it might be an AMD issue indeed.

**Comment by alwinzh** [comment]
Yes，the surface pro also win11.

**Comment by alwinzh** [comment]
The CPU I am using is  AMD Radeon 680M with integrated graphics
The crashing is exactly at the same point as showing in the picture, at the position of "drawing surfaces".

**Comment by oulap** [comment]
@cesargutierrez7
Are you also running simnibs on a computer with AMD? Could you please send the details of your setup, so we can get a better idea of what's going on.

**Comment by cesargutierrez7** [comment]
Yes! These are my specs:

CPU: AMD 7900X
GPU: AMD RADEON 7900XT
RAM: Corsair 32GB KIT

From: oulap ***@***.***>
Date: Thursday, August 10, 2023 at 12:22 PM
To: simnibs/simnibs ***@***.***>
Cc: Cesar Zabdiel Gutierrez Mendoza (czgtrrzm) ***@***.***>, Mention ***@***.***>
Subject: Re: [simnibs/simnibs] SimNibs Crashes when opening ernie folder but not sphere (Issue #158)
CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and trust the content is safe.


@cesargutierrez7<https://github.com/cesargutierrez7>
Are you also running simnibs on a computer with AMD? Could you please send the details of your setup, so we can get a better idea of what's going on.

**Comment by pellet** [comment]
I'm having the same issue, it looks due to the AMD video card I have:
```
Faulting application name: pythonw.exe, version: 3.9.12150.1013, time stamp: 0x624ac72b
Faulting module name: atio6axx.dll, version: 31.0.14057.5006, time stamp: 0x646d3cd7
Exception code: 0xc0000005
Fault offset: 0x000000000027faf3
Faulting process id: 0x3fc8
Faulting application start time: 0x01d9d24bd9d97a74
Faulting application path: C:\Users\beno\SimNIBS-4.0\simnibs_env\pythonw.exe
Faulting module path: C:\WINDOWS\System32\DriverStore\FileRepository\u0393198.inf_amd64_a81d3e409863269f\B392251\atio6axx.dll
Report Id: 70fddcc0-6621-43e2-9e39-a45f5a5ac4c6
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by philippruhnau** [comment]
Is there an update on this? 
I have the same issue for a while on my home pc (and thought it was the old example data sets), but basically I cannot run the simnibs gui as it if freezes when I try to open any of the m2m folders for ernie/MNI. sphere works but takes quite a while. 
I'm running on Win 10 - Intel i7, 32GB ram (which seems to fill up when 'drawing surfaces')

**Comment by LeiGuo0812** [comment]
Same issue on my AMD CPU computer (32 RAM).

**Comment by ayakacxy** [comment]
The same problem appears on my computer with a AMD CPU: 7840HS, but if I use the computer with Intel CPU, this problem doesn't happen.

**Comment by oulap** [comment]
Thanks for reporting this and sorry that we haven't come up with a fix yet. We don't have so many test computers for debugging, and none of the ones we currently have is running an AMD. I'll try to see if we could find one somewhere to reproduce this.

**Comment by ayakacxy** [comment]
I personally believe that the significant differences between AMD's memory management and thread scheduling mechanisms compared to Intel's could be the reason. This might lead to issues such as memory leaks or freezing when running software that is not optimized for AMD.

**Comment by oulap** [comment]
Actually, @ayakacxy  if you're willing to try to install the latest version of simnibs from source using the instructions on the github-page that would be helpful. Maybe building it on an AMD directly would solve this problem, but as said I don't have access to a computer running AMD right now.

The instructions are here:
https://github.com/simnibs/simnibs?tab=readme-ov-file#installation-for-development

If you're building on windows you need to install compilers for the C(++) libraries we compile during the setup.

**Comment by oulap** [comment]
@LeiGuo0812 Thanks for trying! I will try to find a computer with an AMD chip to reproduce this.
